activerecord/lib/active_record/core.rb

```diff
@@ -129,6 +129,23 @@ def self.error_on_ignored_order_or_limit=(value)
 
       mattr_accessor :belongs_to_required_by_default, instance_accessor: false
 
+      ##
+      # :singleton-method:
+      # Indicates whether boolean values are stored in sqlite3 databases as 1
+      # and 0 or 't' and 'f'. If true, boolean columns will store integers 1 and
+      # 0. If false, they will store strings 't' and 'f'. This is false by
+      # default, but using false is deprecated. SQLite databases have used 't'
+      # and 'f' to serialize boolean values and must have old data converted to
+      # 1 and 0 (its native boolean serialization) before setting this flag to
+      # true. Conversion can be accomplshed by setting up a rake task which runs
+      #
+      #   ExampleModel.where(boolean_column: true).update_all(boolean_column: 1)
+      #   ExampleModel.where(boolean_column: false).update_all(boolean_column: 0)
```

Comment:
The example code definitely works, I've used it against a SQLite db.  (Including your suggested modification - I like that it removes the order dependency between running it and setting the flag.)  My concern with providing a gist that generates the code by examining the schema is that it necessarily makes assumptions that I'm not comfortable making.  By providing example code, the responsibility of writing the conversion is with the people who understand their particular configuration, but we give them a good starting point.